Library/Formula/crosstool-ng.rb

```diff
@@ -42,7 +42,7 @@ def install
 
     args << "--with-grep=ggrep" if build.with? "grep"
 
-    args << "--with-make=gmake" if build.with? "make"
+    args << "--with-make=#{HOMEBREW_PREFIX}/bin/gmake" if build.with? "make"
```

Comment:
Sure, but the "make" formula comes from the dupes tap; does it need to be fully qualified, as in `Formula["homebrew/dupes/make"]` ?